index.html

```diff
@@ -135,58 +135,58 @@ <h3 class="article__title">How can I help?</h3>
                 </div>
               </div>
             </div>
-          </div>
-        </div>
+          </section>
+        </main>
 
-        <div class="sidebar col-lg-4">
+        <aside class="sidebar col-lg-4">
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
+          <article class="article article-top">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-spring-fundraisers.jpg"
+              src="images/spring-fundraisers_thumbnail.jpg"
               alt=""
             />
-            <div class="article__body">
+            <article class="article__body article_side-info">
               <h3 class="article__title article__title--sidebar">
                 <a href="#">Join us for our Spring fundraisers</a>
               </h3>
               <p>
                 Riders needed now to help raise awareness and funds for BfR and
                 Médecins Sans Frontières.
               </p>
-            </div>
-          </div>
+            </article>
+          </article>
 
-          <div class="article">
+          <article class="article article-top">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/bikes-for-refugees-logo.jpg"
+              src="images/bikes-for-refugees_logo.jpg"
               alt=""
             />
-            <div class="article__body">
+            <article class="article__body article_side-info">
               <h3 class="article__title article__title--sidebar">
                 <a href="#">Bike dropoff event in Edinburgh on May 11</a>
               </h3>
-            </div>
-          </div>
+            </article>
+          </article>
 
-          <div class="article">
+          <article class="article article-top">
             <img
               class="article__image"
-              src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/news-cycle.jpg"
+              src="images/edinburgh-damascus_thumbnail.png"
               alt=""
```

Comment:
Make sure that there is a description in alt for images, it's important for screen readers

Reply:
Thank you, I forgot to do that with this alt 🙂